src/System.Device.Gpio/System/Device/Pwm/Channels/UnixPwmChannel.cs

```diff
@@ -36,10 +34,10 @@ internal class UnixPwmChannel : PwmChannel
             int frequency = 400,
             double dutyCycle = 0.5)
         {
-            _chip = chip;
-            _channel = channel;
-            _chipPath = $"/sys/class/pwm/pwmchip{_chip}";
-            _channelPath = $"{_chipPath}/pwm{_channel}";
```

Comment:
Another way to do this is just to also take an extra parameter which takes the channel Path format right? that way you don't need the extra properties or change visibility of other members.

Reply:
Yeah. It is a way to do it. I thought about doing it that way actually.
Correct me if I'm wrong, but wouldn't that require a format string like `"pwm-{0}:{1}"`?
That felt less readable than `"pwm-{Chip}:{Channel}"` to me.

I can play with it.

Reply:
I implemented this on another branch.
Personally, I'm not sure how I feel about it.
Your call. I'll bring it in if you think it's preferable.

https://github.com/dotnet/iot/compare/master...rubberduck203:beagle-bone-pwm-alt?expand=1

Reply:
@joperezr any feedback on this?

Reply:
I like the way you changed this now.